docs/using.rst

```diff
@@ -44,17 +24,62 @@ above method instead.
 
 .. _prerequisites:
 
-Prerequisites
-=============
+Installation and Usage
+======================
+
+To install and run the client you just need to type:
+
+.. code-block:: shell
+
+   ./letsencrypt-auto
+
+.. warning:: Please do **not** use ``python setup.py install``.  That mode of
+             operation might corrupt your operating system and is **not supported**
+             by the Let's Encrypt team!
```

Comment:
I very much preferred previous warning message text. It explicitly disallowed certain commands that I've been seeing a lot while supporting clients on IRC. This is rather a step back.